src/TraceEvent/Stacks/SpeedScopeExporter.cs

```diff
@@ -0,0 +1,271 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+
+namespace Microsoft.Diagnostics.Tracing.Stacks
+{
+    public static class SpeedScopeExporter
+    {
+        /// <summary>
+        /// exports provided StackSource to a https://www.speedscope.app/ format 
```

Comment:
Can you also include the links to the documentation on the format that you put in an e-mail to me.